### ISSUE TITLE
Build from source failed during parcel building

### ISSUE THREAD

**Issue by acezzz** [influxdata/chronograf#4341]
I am trying to build from source from the master branch.  But the build process hangs when the parcel script starts a web server.  Any idea?

$ make build
cd ui && yarn run clean && yarn run build
yarn run v1.9.4
$ rm -rf ./build/* && rm -rf ./.cache
Done in 1.06s.
yarn run v1.9.4
$ parcel build -d build --no-source-maps --public-url '' src/index.html
Serving on http://localhost:8080

**Comment by acezzz** [comment]
The code in ui/parcel.js starts the web server. 
By the way, I was able to build the source a while ago when build process was using webpack.

**Comment by russorat** [comment]
@acezzz thanks for reporting. Could you try following the instructions for building in the readme and let us know if you still have this issue? https://github.com/influxdata/chronograf#from-source

**Comment by acezzz** [comment]
@russorat Thanks. I still have the same issue after following the instruction. 
Please note that I am building windows built under cygwin.

My environment:
$ go version
go version go1.10.1 windows/amd64
$ node --version
v8.11.4
$ yarn --version
1.9.4
$ npm --version
6.2.0

The build output is:
$ makego get github.com/influxdata/chronograf
$ cd $GOPATH/src/github.com/influxdata/chronograf
$ make
cd ui && yarn --no-progress --no-emoji
yarn install v1.9.4
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
Done in 167.33s.
cd ui && yarn run clean && yarn run build
yarn run v1.9.4
$ rm -rf ./build/* && rm -rf ./.cache
Done in 0.24s.
yarn run v1.9.4
$ parcel build -d build --no-source-maps --public-url '' src/index.html
Serving on http://localhost:8080
√  Built in 100.29s.

Then the build process will be hanging forever. 

If I just leave as it is, then move to next step. Following is what I am getting
$ go install github.com/influxdata/chronograf/cmd/chronograf
# github.com/influxdata/chronograf/canned
canned\bin.go:20:11: undefined: AssetNames
canned\bin.go:23:18: undefined: Asset
# github.com/influxdata/chronograf/dist
dist\dist.go:39:13: undefined: AssetInfo
dist\dist.go:59:18: undefined: Asset
dist\dist.go:74:11: undefined: Asset
dist\dist.go:83:14: undefined: AssetDir
dist\dist.go:84:14: undefined: AssetInfo

Any idea? Note that I am able to build from the source a while ago when build process was using webpack.

**Comment by acezzz** [comment]
I figured out that the culprit is the naming of ui/parcel.js. It conflicts with node module parcel. So the command "parcel build -d build --no-source-maps --public-url '' src/index.html" actually is trying to run "ui/parcel.js" instead.

My build problem solved after I renamed "ui/parcel.js" to "ui/run_parcel.js" and also updated  one line in parckage.json from
"start": "node parcel.js" 
to 
"start": "node run_parcel.js" 

Thanks for the help.
